Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -23,6 +23,42 @@ public bool IsValid(EntityWhitelist list, [NotNullWhen(true)] EntityUid? uid)
         return uid != null && IsValid(list, uid.Value);
     }
 
+    /// <summary>
+    /// Checks whether a given entity satisfies a whitelist.
+    /// If the whitelist is null it returns null.
+    /// </summary>
+    public bool? Match(EntityWhitelist? list, [NotNullWhen(true)] EntityUid? uid)
```

Comment:
I would honestly rather you just use IsValid checks with extra control flow for null handling because this nullable bool API is extremely weird and nonstandard and i wouldn't want it being used anywhere in the codebase

Reply:
thats just the pre system list?.IsValid, can remove it anyway